### ISSUE TITLE
Segmentation Fault - Error 500 - from Invite a new team member

### ISSUE THREAD

**Issue by cyberjames** [CachetHQ/Cachet#1930]
Before submitting your issue, please make sure that you've checked all of the checkboxes below.

- [x] You're running the [latest release](https://github.com/CachetHQ/Cachet/releases/latest) version of Cachet.
- [x] Ensure that you're running at least PHP 5.5.9, you can check this by running `php -v`
- [x] You've ran `rm -rf bootstrap/cache/*` from the root of your Cachet installation.

To help us better understand your issue, please answer the following — cheers!

### Your setup

- *What version of Cachet?* v2.3.1
- *What database driver? MySQL? Postgres? SQLite?* MySQL
- *What version of PHP?* 5.5.36

### Expected behaviour

*Please describe what you're expecting to see happen.*
Be able to go through to send invitation for new users and no errors of "Segmentation Fault".

### Actual behaviour

*Please describe what you're actually seeing happen.*
After following the complete setup from doc site, when I go to "Team"  and clicking "Invite a new team member" button and filling out email address, when I click submit button, I received "Error 500 Internal Server Error...." 

So I followed what in doc site where I issue command "chmod -R 777 storage" and "chmod -R 777 bootstrap/cache" but still no luck. I tried to refresh and I am getting "Segmentation Fault" at all even from main site.

### Steps to reproduce

*If your issue requires any specific steps to reproduce, please outline them here.*

Go to "Team", click "Invite a new team member" button, put email address and that's it. Then try to refresh the browser and you will be getting segmentation fault errors.

**Comment by GrahamCampbell** [comment]
All segfaults are bugs in PHP itself, not Cachet.

**Comment by cyberjames** [comment]
Thanks for your response. What is the recommended version of PHP to use?

**Comment by GrahamCampbell** [comment]
5.5.37 is the latest in the 5.5.x series. We'd recommend the latest 5.6.x or 7.0.x. That would be 5.6.23 or 7.0.8.

**Comment by devilsoulblack** [comment]
Hi i get this message when ry to load cachet post install


root@status:/var/www/html/Cachet# tail -F /var/log/apache2/error.log
[Wed Jun 29 18:51:56.789008 2016] [core:notice] [pid 801] AH00052: child pid 1420 exit signal Segmentation fault (11)
[Wed Jun 29 18:51:56.789075 2016] [core:notice] [pid 801] AH00052: child pid 1421 exit signal Segmentation fault (11)
[Wed Jun 29 18:52:07.801862 2016] [core:notice] [pid 801] AH00052: child pid 1419 exit signal Segmentation fault (11)
[Wed Jun 29 18:52:07.801932 2016] [core:notice] [pid 801] AH00052: child pid 1423 exit signal Segmentation fault (11)
[Wed Jun 29 18:52:14.810743 2016] [core:notice] [pid 801] AH00052: child pid 1418 exit signal Segmentation fault (11)
[Wed Jun 29 18:52:14.810820 2016] [core:notice] [pid 801] AH00052: child pid 1424 exit signal Segmentation fault (11)
[Wed Jun 29 18:52:22.822528 2016] [core:notice] [pid 801] AH00052: child pid 1415 exit signal Segmentation fault (11)
[Wed Jun 29 18:52:22.822607 2016] [core:notice] [pid 801] AH00052: child pid 1425 exit signal Segmentation fault (11)
[Wed Jun 29 18:52:46.849563 2016] [core:notice] [pid 801] AH00052: child pid 1445 exit signal Segmentation fault (11)
[Wed Jun 29 18:52:46.849646 2016] [core:notice] [pid 801] AH00052: child pid 1446 exit signal Segmentation fault (11)
[Wed Jun 29 18:53:32.902181 2016] [core:notice] [pid 801] AH00052: child pid 1447 exit signal Segmentation fault (11)
[Wed Jun 29 18:53:32.902470 2016] [core:notice] [pid 801] AH00052: child pid 1448 exit signal Segmentation fault (11)


the setup is

What version of Cachet?:  2.3.1
What database driver? : mysql
What version of PHP?; 5.6.22-0+deb8u1

any idea what is the problem ?


![cachet](https://cloud.githubusercontent.com/assets/3916283/16471583/053ae580-3e34-11e6-860f-8777fa1c7cb7.jpg)

**Comment by GrahamCampbell** [comment]
Does it work on PHP 5.6.23?

**Comment by GrahamCampbell** [comment]
Could you install xdebug please? It's possible that an infinite loop is causing the segfault, and xdebug should catch that for us by limiting the callstack size.

**Comment by devilsoulblack** [comment]
@GrahamCampbell how enable xdebug ?

regards

**Comment by GrahamCampbell** [comment]
Without being rude... http://bfy.tw/6Wq8.

**Comment by devilsoulblack** [comment]
@GrahamCampbell thanks

i enable xdebug and get this message over error.log from apache2

http://pastie.org/10895492

any idea what is wrong?

**Comment by GrahamCampbell** [comment]
Right, ok. That does look like an infinite loop.

**Comment by devilsoulblack** [comment]
@GrahamCampbell any idea how to solved ?

**Comment by GrahamCampbell** [comment]
Not quite sure yet. I think it's likely this error is specific to the mysql driver in Laravel.

**Comment by GrahamCampbell** [comment]
Do you have any prefixing or strange database setup going on?

**Comment by devilsoulblack** [comment]
@GrahamCampbell i made the setup part and complete successfull but when try to go Dashboard i get that error on loaad page, i check the databases and is data inside 
![phpmyadmin_cachet](https://cloud.githubusercontent.com/assets/3916283/16503752/d877d7e4-3eea-11e6-8ab9-da566e5e1e68.jpg)

**Comment by cyberjames** [comment]
Any update for php 5.6.22? It is still doesn't work and facing the same issue. Thanks

**Comment by cyberjames** [comment]
After I ran the command "php artisan config:cache", my home page becomes "The Domain page isn’t working". Please help to advise how to fix this?

**Comment by cyberjames** [comment]
After changing mail settings in .env and I ran "php artisan config:cache" command successfully, my homepage went into Internal Server 500 error. This is the log in laravel:

`production.ERROR: exception 'Swift_TransportException' with message 'Cannot send message without a sender address' in /path/to/Cachet/vendor/swiftmailer/swiftmailer/lib/classes/Swift/Transport/AbstractSmtpTransport.php:162
`

Any advise to fix this? Thanks in advanced.

**Comment by GrahamCampbell** [comment]
You need to set a sender address in the mail config. Ping @jbrooksuk. Why isn't this done already?

**Comment by cyberjames** [comment]
@GrahamCampbell, I set "MAIL_ADDRESS=user@domain.com" from my .env already but still no luck.

**Comment by jbrooksuk** [comment]
@GrahamCampbell the mail sender address is set from `MAIL_ADDRESS`?

**Comment by cyberjames** [comment]
Any update guys?

**Comment by jbrooksuk** [comment]
What queue driver are you using @cyberjames?

**Comment by GrahamCampbell** [comment]
Are you using a table prefix in the database?

**Comment by peelman** [comment]
So i had this problem while deploying today.

Turns out, at some point during the setup, my `.env` got polluted; any values that were previously set as `null` got set to the email i gave as my "from" address.  The setup process itself did this; the only things I had populated in that file were my database credentials.

Once I restored null to the database port, database prefix, redis config keys, and others, then re-ran `php artisan config:cache`, the problem went away.

I already left work for the weekend, or I would try to reproduce it.
